examples/with-ant-design/package.json

```
@@ -8,10 +8,8 @@
   },
   "dependencies": {
     "@zeit/next-css": "^1.0.1",
-    "antd": "^3.9.2",
-    "babel-plugin-import": "^1.9.1",
+    "antd": "^4.0.0",
```

maybe you can change the version to 4.0.1, cause they fix a lot bug in this version.

```suggestion
    "antd": "^4.0.1",
```